istio.VERSION

```diff
@@ -1,9 +1,9 @@
 # DO NOT EDIT THIS FILE MANUALLY instead use
 # tests/updateVersion.sh (see tests/README.md)
 export CA_HUB="docker.io/istio"
-export CA_TAG="2017-05-09-17.26.56"
+export CA_TAG="0.1.0"
 export MIXER_HUB="docker.io/istio"
-export MIXER_TAG="2017-05-09-20.54.56"
-export ISTIOCTL_URL="https://storage.googleapis.com/istio-artifacts/stable-621b2c4/artifacts/istioctl"
+export MIXER_TAG="0.1.0"
+export ISTIOCTL_URL="https://storage.googleapis.com/istio-release/releases/alpha-0.1.0/istioctl"
```

Comment:
I was under the impression that the istioctl tool will also be hosted in Github release